plugins/regmap/src/register/bitfield/bitfielddetailedwidget.cpp

```diff
@@ -192,7 +199,7 @@ void BitFieldDetailedWidget::updateValue(QString newValue)
 void BitFieldDetailedWidget::registerValueUpdated(QString newValue)
 {
 	lastReadValue->setText(QString("Current : 0x") + newValue);
-	updateValue(newValue);
+	updateValue(QString("0x") + newValue);
```

Comment:
You should create a function to format values and addresses  (using "0x" for example)
You should also create a function to convert values and addresses to hex and back. Use those functions throughout regmap.